msf_core/include/msf_core/msf_sensorhandler.h

```diff
@@ -52,10 +53,12 @@ class SensorHandler {
         manager_(mng),
         sensorID(constants::INVALID_ID),
         topic_namespace_(topic_namespace),
-        parameternamespace_(parameternamespace) {
+        parameternamespace_(parameternamespace),
+        received_first_measurement_(false) {
   }
   virtual ~SensorHandler() {
   }
+  bool receivedFirstMeasurement() {return received_first_measurement_;}
```

Comment:
I noticed that all (or most) methods in msf are CamelCased not camelCased... I will apply this here as well